platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -779,6 +779,24 @@ public void removeOnSourceChangedListener(OnSourceChangedListener listener) {
     mapChangeReceiver.removeOnSourceChangedListener(listener);
   }
 
+  /**
+   * Set a callback that's invoked when the id of an icon is missing.
+   *
+   * @param listener The callback that's invoked when the id of an icon is missing
+   */
+  public void addOnStyleImageMissingListener(OnStyleImageMissingListener listener) {
+    mapChangeReceiver.addOnStyleImageMissingListener(listener);
+  }
+
+  /**
+   * Set a callback that's invoked when a map source has changed.
+   *
+   * @param listener The callback that's invoked when the source has changed
+   */
+  public void removeOnStyleImageMissingListener(OnStyleImageMissingListener listener) {
```

Comment:
Missing `NonNull` annotation.

Reply:
atm, no map event add/remove have these. Will add them as part of this PR.